content/release-notes/2023-05-09-storage.md

```diff
@@ -0,0 +1,12 @@
+---
+label: 'Storage'
+---
+
+### What's new
+
+- Compute: Docker image files are now included in the [neon](https://github.com/neondatabase/neon) GitHub repository.
```

Comment:
they were here, but, I hope, that now it's possible to use it everyone (we make docker images public)

Reply:
Docker images already were in public Docker Hub and Dockerfiles were also available, but at the same time we are building and pushing them into private registry (ECR). By default, there was private registry used in `FROM ...` clauses in Dockerfile making it impossible for people from outside to run `docker build . -t neondatabase/neon` without knowledge of what should be changed to make it work. I changed defaults to public registry so now `docker build .` will work without any extra access

Reply:
Thank you. Can you take a look at the latest revision?